CHANGELOG.md

```diff
@@ -1,4 +1,24 @@
 ## 3.28.0 (Unreleased)
+
+FEATURES:
+* **New Data Source:** `google_redis_instance` ([#2209](https://github.com/terraform-providers/terraform-provider-google-beta/pull/2209))
+* **New Resource:** `google_notebook_environment` ([#2199](https://github.com/terraform-providers/terraform-provider-google-beta/pull/2199))
+* **New Resource:** `google_notebook_instance` ([#2199](https://github.com/terraform-providers/terraform-provider-google-beta/pull/2199))
+
+IMPROVEMENTS:
+* compute: Added SHARED_LOADBALANCER_VIP as an option for `google_compute_address.purpose` ([#2204](https://github.com/terraform-providers/terraform-provider-google-beta/pull/2204))
```

Comment:
Can this entry be moved to be with its compute neighbors?

Reply:
I forgot my abc's